docs-2.0/nebula-exchange/about-exchange/ex-ug-limitations.md

```diff
@@ -5,6 +5,7 @@
 ## Nebula Graph版本
 
 Exchange client版本（即JAR包版本）和Nebula Graph的版本对应关系如下。
+
 |Exchange client版本|Nebula Graph版本|
 |:---|:---|
 |2.0.0|2.0.0|
```

Comment:
exchange 2.0.0 can be used on both nebula 2.0.0 and nebula 2.0.1.
exchange 2.0.1 is compatible with nebula 2.0.0